### PR TITLE
Feature footer

### PR DESCRIPTION
**Pull request by Ntwali-Josue**
### In this Pull Request I:

- Add the footer section.
- Display dynamically all the footer section contents using JavaScript.
- Style the footer section.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import './style.css';
+
+const root = document.querySelector('.root');
+const footer = document.createElement('footer');
+
+footer.innerHTML = '<h4>Created by Karangwa ntwali Josue and Ángel Díaz under Microverse</h4>';
+
+root.appendChild(footer);
```
Comment (ad9311) [inline src/index.js:8]: As mentioned in the header review it's best to put the footer in the actual index.html with a <footer> tag. This clutters your index.js.

File: src/index.html [hunk 2]
```diff
@@ -8,6 +8,6 @@
 </head>
 <body>
   <div class="root"></div>
-  <script src="index.js"></script>
+  <script src="index.js" type="module"></script>
```
Comment (ad9311) [inline src/index.html:11]: Please remove this, as webpack already does that for you when you compile the project. It is not necessary.

**Review by ad9311 — CHANGES_REQUESTED** [review]
Please check my comments below 👇🏽

File: src/index.html (hunk 2 above)
Comment (Ntwali-Josue) [reply src/index.html]: Thanks for the information 👍🏾 , I did not know about this.

File: src/index.js (hunk 1 above)
Comment (Ntwali-Josue) [reply src/index.js]: Consider it done.

**Comment by Ntwali-Josue** [comment]
**Hey, @ad9311 I have worked on the changes you recently requested. Can you please check again the pull request?**

**Review by ad9311 — APPROVED** [review]
Great work! 👍🏽

**Pull request merged**